### DISCUSSION TITLE
Force slide to end?

### DISCUSSION THREAD

**Discussion by Poylar** [nolimits4web/swiper#5303] [Q&A]
Can I force-scroll the slider? I need the slider to be able to scroll after the end so that there is nothing left

**Comment by htmlcoder001** [comment]
> Can I force-scroll the slider? I need the slider to be able to scroll after the end so that there is nothing left

I think you can take a look at [.slideTo](https://swiperjs.com/swiper-api#method-swiper-slideTo) or [.slideToLoop](https://swiperjs.com/swiper-api#method-swiper-slideToLoop) methods
